### DISCUSSION TITLE
Static generation for part of the pages

### DISCUSSION THREAD

**Discussion by batchor** [nuxt/nuxt#28305] [Questions]
Is there a way to statically generate some pages, like `/about-us` and `/terms-of-use`, during build, while keep other pages server-side or client-side rendered?

Pages like `/about-us` and `/terms-of-use` will never change for a while and I prefer to convert these pages into pure html during building.

**Comment by ghost** [comment]
Take a look at the [Hybrid Rendering](https://nuxt.com/docs/guide/concepts/rendering#hybrid-rendering) section of the docs.
